opengeode/Helper.py

```diff
@@ -608,7 +608,7 @@ def _rename_primcall(ast, from_name, to_name):
 
 
 @rename_everything.register(ogAST.PrimConditional)
-def _rename_ifhthenelse(ast, from_name, to_name):
+def _rename_ifthenelse(ast, from_name, to_name):
```

Comment:
??

Reply:
Name of the function does not matter here (singledispatch is used) but should indeed be fixed.